e2e/testcases/v2-print-certificate/birth/data/birth-declaration.ts

```diff
@@ -0,0 +1,152 @@
l4NjXgCAcFjgiKaAgEChwXK0OHx3MN8vkDgeQHB43k73yRAYHyB9cm1Qsj4NTWDzgUEj84LZHgECFwmkG/F5A2pUQ9vuwxcRzEFBI+YdTdrAgT2BcqTaK2AuFIInCwgeJwMqjkCBKYSWL812CrIVOU1mRYCgkcLdX0SIDCiQN4PYhVkxOoZczcCgkc3pTAQAgQGEbAKMkihDLNPAcGjz7oYFQECYwjYkDpGnYyyIwHBo6NiGAoBAsMKlLdh0iTsBRm2lAZeW0DwqC2sfQIEogk4nCxaxc33kIDgcYjLhwkQIPCwQHk4mQ2pD7P54OwCgsfsFTY/AgRaC+QAkl5Ulw4ncxumdUX031RA8GjKr3MCBIIJlC+q85K6YMU33W8CgocrgQABAtcLpNCRVkDSm3LdhrneX48NBQSPhvi6JkCAwFeBcuXDKohLYnoBwWP6EpsgAQIDCKzfkiuADFA0Q3xOQPB4zs23CBAgUEPAkzA1VLXZlYDg0VU5DIYAAQJvAm6/uBCmFRA8pi2tiREgMIFAuQnV7ZcJCmoKnmpxDRAgQGAEgRQ6Pr2fAyKAjFAxY9wVsOLh4iBAgMA4AuU7YRxENk7djLQQEDxcDgQIEBhLoHwjrvAxVu2M1gFirgECBAgMK5D3fziGfdgSxhy4FY+YdTdrAgTmEch7Puz9mKemU89E8Ji6vCZHgEAQAXs/ghR6hmkKHjNU0RwIECDwTSAFkPRj74crolsBwaPb0hgYAQIEnhbIKyBuvzxN6Iu1BASPWrLaJUCAQFuBFDq8+bZtDfS+ISB4uCwIECAwt4DVj7nrO9zsBI/hSmbABAgQOCwgfBwm84VaAoJHLVntEiBAoD8Bm0/7q0m4EQke4UpuwgQIBBcQPoJfAK2nL3i0roD+CRAg0EZAAGnjHr5XwSP8JQCAAIHgAgJI8Avg6ukLHleL648AAQL9CXj0tr+aTDsiwWPa0poYAQIEDgl48uUQlw8/KyB4PCvnewQIEJhPwDtf5qtpdzMSPLoriQERIECguUA+at3Jp81LMd8ABI/5ampGBAgQOEMgbzpNbXnp3Bmi2ngTEDxcCAQIECBwSyCtfnxYluUjJgJnCAgeZyhqgwABAnML2Psxd30vnZ3gcSm3zggQIDC0gDM/hi5fH4MXPPqog1EQIEBgJAEBZKRqdTZWwaOzghgOAQIEBhHIT77k/x5k2IbZWkDwaF0B/RMgQGBcAeFj3No1G7ng0YxexwQIEJhCIN12+fT1KckfPXY7RT2rT0LwqE6sAwIECEwvkMPHl68zdetl+nK/NkHB4zU/3yZAgACB/wu49eJquCsgeNwl8gECBAgQOCDgZXMHsCJ+VPCIWHVzJkCAQF0B+z7q+g7duuAxdPkMngABAt0KCB/dlqbtwASPtv56J0CAwOwCnz3xMnuJj81P8Djm5dMECBAgcFzASafHzab9huAxbWlNjAABAl0JeOKlq3K0G4zg0c5ezwQIEIgmIHxEq/jGfAUPFwEBAgQIXCkgfFyp3WFfgkeHRTEkAgQITC6QwscvjlifvMo70xM8YtbdrAkQINBawEFjrSvQqH/BoxG8bgkQIEBgyWd9fGQRR0DwiFNrMyVAgECPAsJHj1WpOCbBoyKupgkQIEDgIQGnnD7ENMeHBI856mgWBAgQGF0g7/mw6XT0St4Zv+AxeYFNjwABAoMJpCde8iO3gw3dcB8REDweUfIZAgQIELhSwOO2V2pf3JfgcTG47ggQIEDgIQEHjT3ENN6HBI/xambEBAgQiCLg5XITVlrwmLCopkSAAIGJBNx2maiYaSqCx2QFNR0CBAhMKOCU04mKKnhMVExTIUCAwMQCwsckxRU8JimkaRAgQCCAgPAxQZEFjwmKaAoECBAIJuCsj4ELLngMXDxDJ0CAQGAB4WPQ4gsegxbOsAkQIEDg7YRTR6wPdiEIHoMVzHAJECBA4HcCad9HCh9+BhEQPAYplGESIECAwK6A8DHQxSF4DFQsQyVAgACBm+EjBRAvmOv8IhE8Oi+Q4REgQIDAwwKOWH+Yqt0HBY929nomQIAAgToCnnip43pKq4LHKYwaIUCAAIGOBBw01lEx1kMRPDoujqERIECAwNMCKXx8Wpbl49Mt+GIVAcGjCqtGCRAgQKADAeGjgyJY8eiwCIZEgAABAtUE0n6PD1Y+qvkebtiKx2EyXyBAgACBwQSEj44KJnh0VAxDIUCAAIFqAp/fW7bnoxrxYw0LHo85+RQBAgQIjC+Qw8ePjllvV0zBo529ngkQIEDgeoEUPr587dYJp9fbv/UoeDSC1y0BAgQINBHIT7qkzt12aVACwaMBui4JECBAoKmAx2wb8gseDfF1TYAAAQIEogkIHtEqbr4ECBAgQKChgODREF/XBAgQIEAgmoDgEa3i5kuAAAECBBoKCB4N8XVNgAABAgSiCQge0SpuvgQIECBAoKGA4NEQX9cECBAgQCCagOARreLmS4AAAQIEGgoIHg3xdU2AAAECBKIJCB7RKm6+BAgQIECgoYDg0RBf1wQIECBAIJqA4BGt4uZLgAABAgQaCggeDfF1TYAAAQIEogkIHtEqbr4ECBAgQKChgODREF/XBAgQIEAgmoDgEa3i5kuAAAECBBoKCB4N8XVNgAABAgSiCQge0SpuvgQIECBAoKGA4NEQX9cECBAgQCCagOARreLmS4AAAQIEGgoIHg3xdU2AAAECBKIJCB7RKm6+BAgQIECgoYDg0RBf1wQIECBAIJqA4BGt4uZLgAABAgQaCggeDfF1TYAAAQIEogkIHtEqbr4ECBAgQKChgODREF/XBAgQIEAgmoDgEa3i5kuAAAECBBoKCB4N8XVNgAABAgSiCQge0SpuvgQIECBAoKGA4NEQX9cECBAgQCCagOARreLmS4AAAQIEGgoIHg3xdU2AAAECBKIJCB7RKm6+BAgQIECgoYDg0RBf1wQIECBAIJrA/wBUSN7YXyPllgAAAABJRU5ErkJggg=='
```

Comment:
Could we save the signature as an image file and read it from there, while converting it into base64?